psydac/api/discretization.py

```diff
@@ -225,7 +226,7 @@ def discretize_space(V, domain_h, *args, **kwargs):
         # TODO how to give a name to the mapping?
         symbolic_mapping = Mapping('M', domain_h.pdim)
 
-        if not( comm is None ) and domain_h.ldim == 1:
+        if not( comm is None ) and ldim == 1:
```

Comment:
Instead of writing
```
if not (comm is None)
```
one can use the more common syntax
```
if (comm is not None)
```
or the even shorter
```
if comm
```